.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,12 @@
-# References
-<!-- Any issues or pull requests relevant to this pull request -->
+<!--
+Set the PR title to a meaningful commit message that:
+- follows the Conventional Commits specification (https://www.conventionalcommits.org).
+- is in imperative form.
 
+Template:
+<type>[optional scope]: <description>
+fix: Don't add implicit wildcards ('*') at the beginning and the end of a query (fixes #390).
```

Comment:
I think it's fine to let users figure out the exact format from https://www.conventionalcommits.org listed above, or we should probably explain what an "optional scope" is here as well.
```suggestion
Example:
fix: Don't add implicit wildcards ('*') at the beginning and the end of a query (fixes #390).
```